src/components/Contact/Contact.react.js

```diff
@@ -1,14 +1,13 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import $ from 'jquery';
 import StaticAppBar from '../StaticAppBar/StaticAppBar.react';
 import Footer from '../Footer/Footer.react';
 import './Contact.css';
 
 const Contact = props => {
   document.title =
     'Contact Developer Team of SUSI.AI - Open Source Artificial Intelligence for Personal Assistants, Robots, Help Desks and Chatbots';
-  $('html, body').animate({ scrollTop: 0 }, 'fast');
+  window.scrollTo({ y: 0, behaviour: 'fast' });
```

Comment:
Use **scrollToTopAnimation( )** in `utils/animateScroll`. for reference you can see `src/components/Blog/Blog.react.js`

Reply:
@fragm3 thank you! I will do that